web-console/4.0-designs/storage/object-storage-dashboard/object-storage-dashboard.md

```diff
@@ -0,0 +1,91 @@
+# Object Storage Dashboard￼
+
+## Vision
+Our main goal was to have an object's storage dashboard supported by MCG (NooBaa) that provide the user with at-a-glance information used for analysis and decision making. 
+
+## Object Dashboards
+
+![Object Storage Dashboard](img/object-storage-dashboard.png)
+
+**The objects dashboard includes the following cards:**
+- Details
+- Buckets (Object buckets and Object bucket claims)
+- Resources providers
+- Health
+- Data consumption 
+- Data resiliency 
+- Usage breakdown
+- Data Reduction 
+
+## Details Card
+![Details Crad](img/details-card.png)
+
+**The details card shows:**
+- connected service name and offer external link to the NooBaa (MCG) system. 
+- Provider name
+- MCG installed version
+
+## Buckets Card 
+![Buckets Crad](img/buckets-card.png)
+
+MCG - MCG/NooBaa Buckets are data containers that behave similarly to AWS S3 buckets. Users can set policies that will determine which resources will be user (per bucket) to store data.
+We will show a breakdown between static buckets and buckets claim. Each bucket type will have a status indication when something is wrong with X buckets (shows the number of buckets that are currently in warning/ error status). The number link should lead to the problematic buckets list. 
```

Comment:
Right now the "Down arrow" icon with the blue link doesn't look like a "Problem".
Should that link be red? maybe a label?

Reply:
@mceledonia @andybraren - Thoughts on this from a consistency standpoint with the Overview Dashboard?

Reply:
@yuvalgalanti IIRC we discussed changing these icons to the regular error/warning icons, right? I don't know anything about buckets, but even if "down" is what the status is technically called, an error icon with "Down" as the label would probably be clearer here. I would expect the same icon and label to be used in a future Buckets List page (if that page already exists, the "12 Object Buckets" label should also be blue here).